test/extensions/filters/common/rbac/matchers_test.cc

```diff
@@ -192,32 +192,53 @@ TEST(AuthenticatedMatcher, uriSanPeerCertificate) {
   Envoy::Network::MockConnection conn;
```

Comment:
Add test where name is not set, only the new principal_name field is.

Reply:
Done